config/config-example.js

```diff
@@ -73,10 +75,17 @@ exports.reportjoins = true;
 //   getting more than 160 or so users.
 exports.reportbattles = true;
 
-// report joins and leaves in battle - shows messages like "<USERNAME> joined" in battle
-// Turn this off on large tournament servers where battles get a lot of joins and leaves.
+// report joins and leaves in battle - shows messages like "<USERNAME> joined"
+//   This feature can clutter battle logs - turn this off if your server hosts
+//   battles with more than 30 or so spectators (such as big tournament battles).
 exports.reportbattlejoins = true;
 
+// report joins and leaves periodically - sends silent join and leave messages in batches
+//   This setting will only be effective if `reportjoins` is set to false, and users will
+//   only be able to see the messages if they have the /showjoins client-side setting enabled.
+//   Set this to a positive amount of milliseconds if you want to enable this feature.
+exports.reportjoinsperiod = 0;
+
```

Comment:
They're not supposed to be seen at all.

The batches are to save bandwidth, it's not supposed to have any effect on how they're seen since reportjoins is supposed to be off.